units_cpp/src/units/headers/units/constants/earth_rotation.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * earth_rotation.h
+ *
+ *  Created on: Nov 3, 2017
+ *      Author: oriash93
+ */
+
+#ifndef SRC_UNITS_HEADERS_UNITS_CONSTANTS_EARTH_ROTATION_H_
+#define SRC_UNITS_HEADERS_UNITS_CONSTANTS_EARTH_ROTATION_H_
+
+#include <units/angular_speed.h>
+
+namespace units {
+namespace earth_rotation
```

Comment:
too specific. we want this name to be a GROUP name, so it could contain other constants too. what about "planetary_rotations"?